sencha-workspace/packages/slate-cbl/src/view/student/skill/OverviewWindow.js

```diff
@@ -0,0 +1,85 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.student.skill.OverviewWindow', {
+    extend: 'Ext.window.Window',
+    xtype: 'slate-cbl-student-skill-overviewwindow',
+    requires: [
+        'Slate.cbl.view.student.skill.OverviewWindowController',
+        'Slate.cbl.view.skill.OverviewBody',
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    controller: 'slate-cbl-student-skill-overviewwindow',
+
+    config: {
+        student: null,
+        competency: null,
+        skill: null
+    },
+
+    title: 'Skill Overview',
+    width: 700,
+    minWidth: 700,
+    // constrainHeader: true,
```

Comment:
I believe because it caused problems when these modal windows started getting bigger than the browser viewport, it prevents the modal window header from being dragged out of the browser viewport. It was left here though because it was still sort of an open question how to best deal with them growing in height